src/Psecio/Versionscan/Command/MissingCommand.php

```diff
@@ -33,23 +35,46 @@ protected function configure()
     protected function execute(InputInterface $input, OutputInterface $output)
     {
         $this->verbose = $input->getOption('verbose');
-        $changelog = file_get_contents('http://php.net/ChangeLog-5.php');
+        $this->checksFilePath = __DIR__ . '/../../../Psecio/Versionscan/checks.json';
+        $saveResults = $input->getOption('save-results');
 
         // Get our current checks
-        $json = json_decode(file_get_contents(__DIR__.'/../../../Psecio/Versionscan/checks.json'));
-        $checksList = [];
-        foreach ($json->checks as $check) {
-            if (!in_array($check->cveid, $checksList)) {
-                $checksList[] = $check->cveid;
+        $this->checksFileContents = json_decode(file_get_contents(__DIR__.'/../../../Psecio/Versionscan/checks.json'), true);
```

Comment:
Can this reference `$this->checksFilesPath` instead?

Reply:
Fixed, that was always the intent, but I forgot to follow through